webclient/app/src/app/features/decision/DecisionOverviewMap.jsx

```diff
@@ -39,16 +43,25 @@ function DecisionOverviewMap() {
         reloadDecisions();
         const interval = setInterval(reloadDecisions, 5000); // Update alle 5 Sekunden
         return () => clearInterval(interval);
-    }, []);
+    }, [selectedType]);
 
     //Load Decisions
     function reloadDecisions() {
-        decisionRest.findAllOpen().then(response => {
-            if (response.data) {
-                setDecisions(response.data);
-            }
-        });
+        if (selectedType === 'all') {
```

Comment:
As you already have loaded all open types, it might be better to filter result in UI